unittests/AST/ASTImporterGenericRedeclTest.cpp

```diff
@@ -666,11 +755,127 @@ ASTIMPORTER_INSTANTIATE_TYPED_TEST_CASE(RedeclChain, FunctionTemplateSpec, ,
                                         ImportPrototypeThenProtoAndDefinition)
 
 ASTIMPORTER_ODR_INSTANTIATE_TYPED_TEST_CASE(RedeclChain, Class, Conservative, ,
-                                            DoNotImportConflictingDefinition)
-ASTIMPORTER_ODR_INSTANTIATE_TYPED_TEST_CASE(RedeclChain, Class, Liberal, ,
-                                            ImportConflictingDefinition)
+                                            DontImportConflictingDefAfterDef)
 ASTIMPORTER_ODR_INSTANTIATE_TYPED_TEST_CASE(RedeclChain, Class, Liberal, ,
-                                            ImportConflictingDefinitionFromTU)
+                                            ImportConflictingDefAfterDef)
+
+ASTIMPORTER_ODR_INSTANTIATE_TYPED_TEST_CASE(
```

Comment:
Would it be possible to format these lines with the same padding? The goal is to be able to look through easily the group of tests.
I know that clang-format will not do that properly, so we may use `// clang-format off` and `// clang-format on` comments (https://clang.llvm.org/docs/ClangFormatStyleOptions.html).
I am thinking about something like this:
```
// clang-format off
// ========
// Variable
// ========
ASTIMPORTER_ODR_INSTANTIATE_TYPED_TEST_CASE(RedeclChain, Variable, 
  Conservative, ,DontImportConflictingProtoAfterProto)
ASTIMPORTER_ODR_INSTANTIATE_TYPED_TEST_CASE(RedeclChain, Variable, 
  Conservative, ,DontImportConflictingDefAfterDef)
ASTIMPORTER_ODR_INSTANTIATE_TYPED_TEST_CASE(RedeclChain, Variable,
  Conservative, ,DontImportConflictingProtoAfterDef)
... 

// =============
// ClassTemplate
// =============
...
// clang-format on
```